packages/pds/src/api/com/atproto/sync/getRepoStatus.ts

```diff
@@ -0,0 +1,41 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { assertRepoAvailability } from './util'
+import { AccountStatus } from '../../../../account-manager'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.sync.getRepoStatus({
+    handler: async ({ params }) => {
+      const { did } = params
+      const account = await assertRepoAvailability(ctx, did, true)
+      const active = !account.takedownRef && !account.deactivatedAt
+
+      let status: string | undefined = undefined
+      if (!active) {
+        if (account.takedownRef) {
+          status = AccountStatus.Takendown
+        } else if (account.deactivatedAt) {
+          status = AccountStatus.Deactivated
+        }
+      }
```

Comment:
I would almost flip these around:
```suggestion
      let status: string | undefined = undefined
      if (account.takedownRef) {
        status = AccountStatus.Takendown
      } else if (account.deactivatedAt) {
        status = AccountStatus.Deactivated
      }

     const active = !status
```

Reply:
yeah i dig 👍